addons/sale/models/payment_transaction.py

```diff
@@ -125,8 +125,11 @@ def _reconcile_after_done(self):
             self._invoice_sale_orders()
         super()._reconcile_after_done()
         if auto_invoice:
-            # Must be called after the super() call to make sure the invoice are correctly posted.
-            self._send_invoice()
+            if str2bool(self.env['ir.config_parameter'].sudo().get_param('sale.async_emails')):
+                self.env.ref('sale.send_invoice_cron')._trigger()
```

Comment:
Let's just make sure that the cron exists and can be run.
```suggestion
            async_send = str2bool(self.env['ir.config_parameter'].sudo().get_param('sale.async_emails'))
            send_invoice_cron = self.env.ref('sale.send_invoice_cron', raise_if_not_found=False)
            if async_send and send_invoice_cron:
                if not send_invoice_cron.active:
                    send_invoice_cron.active = True
                self.env.ref('sale.send_invoice_cron')._trigger()
```